bin/reth/src/args/miner_args.rs

```diff
@@ -0,0 +1,31 @@
+use reth_primitives::Address;
+
+#[derive(Debug, Args, PartialEq, Default)]
+pub struct MinerArgs {
+    #[arg(long = "mine", help_heading = "Miner", default_value = false)]
+    pub mine: bool,
+    #[arg(long = "miner.etherbase", help_heading = "Miner", default_value = "0x000000000")]
```

Comment:
I'm not sure the default value works, because too short.

please add a sanity parse test for this type, see RpcServerArgs test for reference